routers/posts.router.js

```diff
@@ -0,0 +1,192 @@
+const express = require("express");
+const router = express.Router();
+const { authVerify } = require("../middleware/authVerify");
+const { Post } = require("../models/post.model");
+const { User } = require("../models/user.model");
+
+router.use(authVerify);
+
+router.route("/").post(async (req, res) => {

```

Comment:
route functions can be moved to a controllers folder and be imported here so that your routes file is more clean and readable